From: Filipe Brandenburger <filbranden@google.com>
Date: Wed, 27 Jun 2018 09:19:33 -0700
Subject: sleep: fix one more printf format of a fiemap field

Use PRIu64 constant to get the format right on LP-64 architectures,
cast to (uint64_t) to solve incompatibility of __u64.

This was missed in ad4bc3352285f467f4ffa03c3171b19fa0a8758d, so fix it
with this follow up.

(cherry picked from commit d7af62d52cd3b58c5fe7d410e907e01fed71c02e)
---
 src/test/test-sleep.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/test/test-sleep.c b/src/test/test-sleep.c
index acfcf6a..a8ad302 100644
--- a/src/test/test-sleep.c
+++ b/src/test/test-sleep.c
@@ -39,8 +39,8 @@ static int test_fiemap(const char *path) {
         log_info("\t number of mapped extents: %" PRIu32, fiemap->fm_mapped_extents);
         log_info("\t extent count: %" PRIu32, fiemap->fm_extent_count);
         if (fiemap->fm_extent_count > 0)
-                log_info("\t first extent location: %llu",
-                         fiemap->fm_extents[0].fe_physical / page_size());
+                log_info("\t first extent location: %" PRIu64,
+                         (uint64_t) (fiemap->fm_extents[0].fe_physical / page_size()));
 
         return 0;
 }
